static/docs/commands-reference/repro.md

```diff
@@ -57,6 +57,11 @@ specified), and updates stage files with the new checksum information.
   reproduced as part of the pipeline in the parent directory, or as an
   independent unit.
 
+- `-R`, `--recursive` - `targets` is expected to contain directory path(s).
+  Determines the stages to reproduce by searching each target directory and its
+  subdirectories for DVC-files to inspect. Along with providing `targets`, this
```

Comment:
the last sentence sounds strange. Should we change it to something like: `Along with --with-deps and other similar options it is a way to set the scope (DVC-files) to reproduce`?

Reply:
Actually in my latest update to the -R option explanation I thought about it and decided to remove it as recursiveness does not limit any scope, if anything it expands the number of DVC-files that are checked or produced.